test/fixtures/multifile/multifile_hooks.coffee

```diff
@@ -1,10 +1,10 @@
-{after} = require 'hooks'
+{after, log} = require 'hooks'
```

Comment:
I'm strictly against modifying existing fixtures and tests especially when this functionality should preserved. Please create separate fixture utilising the `hooks.log` and leave this `console.log` one for backward compatibility,